src/ensembl/src/content/app/browser/track-panel/trackPanelState.ts

```diff
@@ -21,8 +21,8 @@ import pick from 'lodash/pick';
 export type PreviouslyViewedObject = {
   genome_id: string;
   object_id: string;
-  object_type: string;
-  label: string;
+  type: string;
+  label: (string | null)[];
```

Comment:
Why did you prefer it over the type `string | string[]` (string or array of strings?)? And what's with the `null[]` type?

Reply:
I think you are right. Label is never going to be null. It will be either symbol + stableId or just stableId.